STAGED_UPGRADE_PLAN.md

```
@@ -0,0 +1,1474 @@
+```
+
+---
+
+### Step 1.2: Update auth_store.go Imports
+
+**File:** `auth_store.go`
+
+**Current (Lines 3-6):**
+```go
+import (
+	"github.com/jinzhu/gorm"
```

markdownlint-cli2 (0.18.1)

Hard tabs
Column: 1

(MD010, no-hard-tabs)